common/mouse.go

```diff
@@ -199,7 +199,7 @@ func (m *Mouse) Up(x float64, y float64, opts goja.Value) {
 		k6ext.Panic(m.ctx, "parsing up options: %w", err)
 	}
 	if err := m.up(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "mouse up: %w", err)
+		k6ext.Panic(m.ctx, "releasing mouse up: %w", err)
```

Comment:
`"releasing mouse up"` sounds redundant. If we're releasing a mouse click, it's a "mouse up" event. So maybe this should be just `"releasing mouse"`.

But I'm slightly against these mouse error changes. E.g. `"moving mouse"` is slightly wrong; the cursor/pointer is what moves. So I'd prefer keeping these as they were, since they just describe the actual `Mouse` method that was called, and are less confusing. Even if it goes against our new conventions.

Reply:
Earlier, I suggested using "releasing mouse", then you were against it. That's why I combined both. I can make it "releasing the mouse button" again. It's universal that everyone can understand that releasing the mouse button means mouse up.

We can say "moving the mouse pointer" instead of ditching them all.

Reply:
OK, let's go with `"releasing the mouse button"` and `"moving the mouse pointer"` then. :+1: